src/styles/cards.scss

```diff
@@ -0,0 +1,139 @@
+html {
+  font-family: $catalog-font-family;
+  font-weight: $catalog-font-weght;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.main__container {
+  margin: 0 auto;
+  width: (4 * $card-width) + (3 * $card-margin) + (2 * $inside-padding);
+  display: flex;
+  flex-wrap: wrap;
+  padding-left: 2px;
+}
+
+p {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+  color: $white;
+}
+
+.card {
+  background-color: var(--white-color);
+  max-width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 0;
+  cursor: pointer;
+  margin: 0 24px 48px ;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: flex;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-style: normal;
+    font-size: 12px;
+    line-height: 18px;
+    color: $dark-blue;
+    margin-bottom: 4px;
+  }
+
+  &__product {
+    font-family: inherit;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $card-color;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__text {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $card-color;
+  }
+
+  &__cost {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  &__button {
+    display: block;
+    width: 100%;
+    background-color: $aqua;
+    text-align: center;
+    line-height: 38px;
+    border: 1px solid $aqua;
+    border-radius: 5px;
+    font-weight: 700;
+    font-size: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__button:hover {
+    background-color: transparent;
+    color: $aqua;
+  }
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-child(-n + 4) {
+    margin-top: 48px;
+  }
+
+}
+
+.stars {
+  display: flex;
+  margin-bottom: 24px;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background: url(../images/star.svg) center no-repeat;
+    margin-right: 4px;
+  }
+
+  &--1 &__star:nth-child(1),
+  &--2 &__star:nth-child(-n+2),
+  &--3 &__star:nth-child(-n+3),
+  &--4 &__star:nth-child(-n+4),
+  &--5 &__star:nth-child(-n+5) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &__reviews {
+    font-family: $catalog-font-family;
```

Comment:
if you set font family in html selector - there is no need to specify it again in elements

Reply:
> if you set font family in html selector - there is no need to specify it again in elements

usually )) at the beginning, I set the values ​​in the elements, then I understand that I need to specify in the html selector, and I forget to remove the old values )